packages/react-components/lib/error-overlay.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+import { Typography, Grid } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  errorIcon: {
+    color: theme.palette.error.main,
+    fontSize: '2rem',
+  },
+  errorMsg: {
+    margin: '0.5rem',
+  },
+  errorDisabled: {
+    pointerEvents: 'none',
+    filter: 'blur(.25rem)',
+    gridArea: '1 / 1',
+    opacity: 0.6,
+  },
+  overlay: {
+    gridArea: '1 / 1',
+    backdropFilter: 'blur(.5rem)',
+    padding: theme.spacing(1),
+  },
+  container: {
+    display: 'grid',
+  },
+  disableSelect: {
+    userSelect: 'none',
+  },
+}));
+
+export interface ErrorOverlayProps {
+  errorMsg?: string | null;
+  children: JSX.Element | null;
+  overrideErrorStyle?: string;
+}
+
+export const ErrorOverlay = React.memo(
+  (props: ErrorOverlayProps): JSX.Element => {
+    const classes = useStyles();
+    const { errorMsg, children, overrideErrorStyle } = props;
+
+    return errorMsg ? (
+      <React.Fragment>
```

Comment:
We don't need `React.Fragment` here as the `div` already encapsulates everything